data/json/recipes/food/offal_dishes.json

```diff
@@ -367,5 +367,50 @@
       [ [ "pepper", 3 ] ],
       [ [ "milk_standard", 1, "LIST" ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "sausage_casings",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": "50 m",
+    "batch_time_factors": [ 83, 5 ],
+    "charges": 60,
+    "byproducts": [ [ "ruined_chunks", 2 ] ],
+    "//": "Charcuterie: The Craft of Salting, Smoking, and Curing would be a good book for someone to use for further meat products recipes.",
+    "book_learn": [ [ "offalcooking", 3 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 }, { "id": "CUT_FINE", "level": 1 } ],
+    "tools": [ [ [ "funnel" ] ] ],
+    "components": [ [ [ "offal", 2 ] ], [ [ "salt", 20 ] ], [ [ "water", 2 ] ], [ [ "bag_plastic", 1 ] ] ]
```

Comment:
As far as I can tell, offal can't be harvested from corpses.
It should probably be obsoleted.

Reply:
Think I should just change that to stomachs then?  Despite casings coming from the lower intestines?  I can make another PR to obsolete offal since it's been changed into discrete harvest items.

Reply:
It's looks like it's been used interchangeably as a harvest type, an item, a component list.  Lots of things "looks_like" offal.  :(

Reply:
You can also add lower intestines to harvest lists and use those instead.

Reply:
Discrete organs get turned into offal when canned, which then can be used in various recipes, including in an unfortunate turn of events "canned_offal".